### PR TITLE
nolo tlc ciel 9_6C to 9_6_1

### PR DESCRIPTION
**Pull request by nolo16**

### CODE REVIEW

File: script/Ciel/Day 9/09_01_CIEL09_6C.txt [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
 -- Page 10023, Offset 23833.
 -- 先輩は俺にそう呼びかけながら、強風のようなアルクェイドの突進を避けていた。
-Senpai warns me while avoiding Arcueid's whirlwind charge.
+Senpai warns me while avoiding Arcueid's tempestuous charge.
 }
 [sha:2577914e7c23d8c567d811dfb4006a8cfdd69537]{
 -- Page 10023, Offset 23834.
 -- 跳び退いた先は広場の<隅|すみ>……あきらかに俺を逃がすために距離を取っている。
-She leaps back to a corner of the park, very clearly moving further away so as to let me escape.
+She leaps back to a corner of the park. She's clearly trying to build some distance so that I can escape.
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_6C.txt:14]: build some distance to let me escape

File: script/Ciel/Day 9/09_01_CIEL09_6C.txt [hunk 2]
```diff
@@ -162,17 +163,17 @@ Ah."
...
 -- Page 665, Offset 23865.
 -- 理性が働かない―――それがどうした。
-My sense of reason is not working―――What about it?
+My sense of reason isn't working―――So what?
 }
 [sha:84b8abd47d07cc5309997c1bb0c5a0ba208e08c1]{
 -- Page 665, Offset 23866.
 -- アルクェイドは容赦なく先輩を殺すだろう。
-Arcueid will probably show no mercy to Senpai and kill her.
+Arcueid will probably mercilessly kill Senpai.
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_6C.txt:176]: ### DO IT!

THERE CAN BE ONLY ONE

File: script/Ciel/Day 9/09_01_CIEL09_6_1.txt [hunk 3]
```diff
@@ -6,17 +6,17 @@
 [sha:cbfa097a63895cb98f8b42905d4302f8750e60b8]{
 -- Page 10024, Offset 23869.
 -- そんな馬鹿げた事を、このまま見過ごせる筈がない……！
-I'm certainly not going to let her do something so stupid without doing anything!
+I absolutely refuse to just stand by and watch as all this stupid crap unfolds before my eyes!
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_6_1.txt:9]: weakling

File: script/Ciel/Day 9/09_01_CIEL09_6_1.txt [hunk 4]
```diff
@@ -101,17 +101,17 @@ But, even so―――
...
 -- Page 675, Offset 23889.
 -- 「聞きたくないっ！　退いて、志貴！」
-"That's not what I want to hear! Out of my way, Shiki!"
+"That's not what I wanted to hear! Out of my way, Shiki!"
 }
 [sha:a7cf80845f0fa9ed3cbb6c5ba98e469cfaeb8c43]{
 -- Page 676, Offset 23890.
 -- 「おまえこそ退けアルクェイド……！　おまえは俺に言ったじゃないか、人間は殺さないって！　それとも、あれは俺を安心させる為の嘘だっていうのか……！」
-"You get out of my way, Arcueid! Wasn't if you who told me that you don't kill humans?! Or was that a lie to get me to trust you?!"
+"You get out of my way, Arcueid! You were the who told me that you didn't kill humans, remember?! Or were you just lying to me so that you could earn my trust?!"
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_6_1.txt:114]: just lying to earn my trust?!

**Review by Avuxo — APPROVED** [review]
better end: shiki helps arc kill ciel then they bang

File: script/Ciel/Day 9/09_01_CIEL09_6C.txt (hunk 2 above)
Comment (nolo16) [reply script/Ciel/Day 9/09_01_CIEL09_6C.txt:176]: 😨

**Pull request merged**